blog/2022-09-30-mdx-blog-post.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: getting-started
```

Comment:
Does this needs to be kebab case?

Reply:
I'll leave it as kebab-case for now. @jacoobes reply to this if you want anther case type